testsuite/classlibrary/TestNodeProxy.sc

```diff
@@ -235,4 +235,14 @@ TestNodeProxy : UnitTest {
 		this.assertEquals(oldProxy.nodeMap, newProxy.nodeMap);
 	}
 
+	test_copy_independent {
+		var copied;
+		proxy.source = 1967;
+		copied = proxy.copy;
+		copied.source = 1968;
+		this.assertEquals(proxy.source, 1967, "copying a proxy should return an independent object");
+	}
+
```

Comment:
And here? At the end of a class, I usually just put the closing braces on consecutive lines.

Reply:
same, I always maintain newlines between methods. It looks much tidier to me.

Reply:
ah I see, it's duplicate, sorry.